Core/Config.cpp

```diff
@@ -605,6 +605,8 @@ static ConfigSetting graphicsSettings[] = {
 #ifdef _WIN32
 	ConfigSetting("D3D11Device", &g_Config.sD3D11Device, "", true, false),
 #endif
+	ConfigSetting("VendorChecksEnabled", &g_Config.bVendorChecksEnabled, true, false, false),
```

Comment:
Maybe this could be more clearly `VendorBugWorkarounds` or `VendorBugChecksEnabled`?

-[Unknown]

Reply:
Yeah, that's probably a better name.